tests/EmailAddressTest.php

```diff
@@ -140,4 +151,33 @@ public function equalityTestEmailAddresses(): array
             [new EmailAddress('fake@example.com'), new EmailAddress('FAKE@example.com'), true, true],
         ];
     }
+
+    /**
+     * @return mixed[][]
+     */
+    public function hasTechnicalRoleAliasDataProvider(): array
+    {
+        return [
+            // Non technical roles
+            [new EmailAddress('fake@example.com'), false],
+            [new EmailAddress('info@example.com'), false],
+            [new EmailAddress('marketing@example.com'), false],
+            [new EmailAddress('sales@example.com'), false],
+            [new EmailAddress('support@example.com'), false],
+            [new EmailAddress('hostmaster@example.com'), false],
+            [new EmailAddress('usenet@example.com'), false],
+            [new EmailAddress('news@example.com'), false],
+            [new EmailAddress('webmaster@example.com'), false],
+            [new EmailAddress('www@example.com'), false],
+            [new EmailAddress('uucp@example.com'), false],
+            [new EmailAddress('ftp@example.com'), false],
+            // Technical roles
+            [new EmailAddress('abuse@example.com'), true],
+            [new EmailAddress('noc@example.com'), true],
+            [new EmailAddress('security@example.com'), true],
+            [new EmailAddress('postmaster@example.com'), true],
+            // Test uppercase values.
+            [new EmailAddress('FTP@example.com'), true],
```

Comment:
Shouldn't this expect `false`? I'm guessing you had added `ftp` to the aliases and then removed it. We should probably just change it to `PostMaster@example.com`.